lib/Server.js

```diff
@@ -832,6 +832,9 @@ class Server {
       if (proxyConfig.target) {
         return createProxyMiddleware(context, proxyConfig);
       }
+      this.logger.warn(
+        `The "target" value specified in the "proxy" property ${context} is not defined and will be skipped.`
```

Comment:
for better visibility 

```suggestion
        `The "target" value specified in the "proxy" property "${context}" is not defined and will be skipped.`
```

Can you also add a simple test case in https://github.com/webpack/webpack-dev-server/blob/master/test/server/proxy-option.test.js ?